tests/conftest.py

```diff
@@ -328,7 +328,7 @@ def framework_env(pytestconfig):
         TESTS_DATA_PATH, "10k_diabetes_sample.csv"
     ),
     (None, MULTICLASS_LABEL_SPACES): os.path.join(TESTS_DATA_PATH, "iris_with_spaces_full.csv"),
-    (SKLEARN, TEXT_GENERATION): os.path.join(TESTS_DATA_PATH, "prompts.csv"),
+    (PYTHON_TEXT_GENERATION, TEXT_GENERATION): os.path.join(TESTS_DATA_PATH, "prompts.csv"),
```

Comment:
why is this renamed?

Reply:
Because we define dataset for the (framework, problem) pair.
And it was just defined wrong.